lib/ignored-paths.js

```diff
@@ -236,13 +236,16 @@ class IgnoredPaths {
 
         let result = false;
         const absolutePath = path.resolve(this.options.cwd, filepath);
-        const relativePath = pathUtil.getRelativePath(absolutePath, this.baseDir);
 
         if ((typeof category === "undefined") || (category === "default")) {
+            const relativePath = pathUtil.getRelativePath(absolutePath, this.options.cwd);
```

Comment:
I think this should only apply when `--ignore-path` is explicitly passed on command line, otherwise it should resolve based on the location of the `.eslingignore` file. Documentation states that it would be resolved based on the location of the file.

Reply:
If no `ignore-path` option is passed on the command line, then the `.eslintignore` file is always in the CWD anyway, so the paths would be resolved the same either way.

Reply:
Hmm... Good point.

Reply:
Thanks for the discussion. Any tips on what I should do to test this? Do I change the existing tests (the ones breaking due to this change)? I would expect that to be a breaking change technically 🤔